### ISSUE TITLE
Normalization

### ISSUE THREAD

**Issue by sseliner** [BenediktAlkin/upt-tutorial#1]
First thanks a lot for this interesting work.

I have a question regarding the normalization for CFD-Simulations like in the SimulationDatatset. I thought the mean and std values are generated like in the [upt-code-base](https://github.com/ml-jku/UPT) with the script [cfddataset_norm.py](https://github.com/ml-jku/UPT/blob/main/data/transientflow/cfddataset_norm.py), but I think that is not the case.
I also tried this normalization with my own CFD-Data but then the model does not work as intended. If I insted use the same values for mean and std like in the SimulaitonDataset it works quite well. 

Could you please explain how the normalization is done in more detail?

From my observations, the model appears to be highly sensitive to the normalization values. Is there a reason for this?

**Comment by BenediktAlkin** [comment]
Very good catch!

It seems like I copied in the wrong normalization values for this tutorial (I used the values for "v1-10000sims" of the [UPT codebase](https://github.com/ml-jku/UPT/blob/main/src/datasets/cfd_dataset.py#L97), which are the normalization values for a dataset that we generated during early development of UPT, which turned out to have some errors in the simulation setup).

For the results reported in the UPT paper, we use the ["mean0std1q25" norm values of the "v3-10000sims" dataset version](https://github.com/ml-jku/UPT/blob/main/src/datasets/cfd_dataset.py#L141), which were created with the `cfddataset_norm.py` script (we chose the inter-quartile range normalization because the dataset contains outlier values; for more info see Appendix D.5 in the paper). This should have also been used for as normalization values for this tutorial.

It's probably not too important for the tutorial, as the tutorial showcases "overfitting on a single short trajectory", but you are absolutely right that normalization can be a vital hyperparameter. We typically use normalization to approximately mean=0 std=1 and apply a logscale if the dataset contains outlier values that lead to training instabilities. 

We have not studied sensitivity to data normalization in detail, but I'd guess that the reason is that physics data can be quite tricky to process. You often need high numerical precision (e.g., training in fp32), which could also be the reason why a model is sensitive to normalization values. This is somewhat contrary to other fields such as computer vision, where models can leverage mixed-precision training (e.g., fp16) without any performance loss and the models are less sensitive to normalization because the value range is nicely bounded to a [0, 1] range for natural images.
